arrow.sh

```diff
@@ -99,19 +100,21 @@ cmake ./src_tmp/cpp
       ${UTF8PROC_ROOT:+-Dutf8proc_ROOT="$UTF8PROC_ROOT"}                                            \
       ${OPENSSL_ROOT:+-DOpenSSL_ROOT="$OPENSSL_ROOT"}                                               \
       ${CLANG_ROOT:+-DLLVM_DIR="$CLANG_ROOT"}                                                       \
+	  ${PYTHON_ROOT:+-DPython3_EXECUTABLE="$PYTHON_ROOT/bin/python3"}								\
```

Comment:
Please, align things as they already are.

Reply:
Oh my bad, I didn't notice that I was aligning with tabs